debugger/src/main/resources/acceptable-security-check-failures.txt

```diff
@@ -16,3 +16,9 @@ org\.apache\.pdfbox\.pdmodel\.font\.FileSystemFontProvider.*
 # continue with a valid default for Windows
 java\.io\.FilePermission ".*somaxconn", "read"
 io\.netty\.util\.NetUtil\$.*:267
+
+# At startup time, Karaf tries to create a system terminal and will fall back to a dumb one of it
```

Comment:
There one type. The word "of" should be "if", here:

expected: "**dumb one it it**"
actual: "dumb one of it"